packages/developer-extension/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Since this extension will not be published as an npm package, I would move it out of the `packages` directory. What about `extensions/chrome`? That way if we have other browser extensions in the future we can store them into the `extensions` directory too.

Reply:
I'm fine with moving the extension in another folder (@bcaudan WDYT?), but I wouldn't call the folder `chrome` since we should not need a separate extension for each browsers (Firefox and Safari are using WebExtensions too, so they can have the same codebase)

Reply:
I'm fine with both, I have a slight preference for having it outside of packages too, since it is not sdk code (a bit like e2e tests code).
I agree that chrome is not a suitable name.